src/config/string_list_format.js

```diff
@@ -0,0 +1,17 @@
+
+module.exports = {
+  name: 'string-list',
+  validate (val) {
+    if (!(val instanceof Array)) {
+      throw new Error('must be an Array')
+    }
+    val.forEach(item => {
+      if (typeof item !== 'string') {
+        throw new Error('every value in list must be a string')
+      }
+    })
+  },
+  coerce (val) {
+    return val ? val.split(',') : []
+  }
+}
```

Comment:
Rather than rolling our own string parsing, I think we can use the [built in](https://github.com/mozilla/node-convict/tree/master/packages/convict#custom-format-for-array-items) functionality in `convict`

Reply:
This looks nice, but I don't see how I can apply it to an environment variable (string) that I want to be interpreted as an array. Do you have an idea in mind?